### PR TITLE
handling empty PM list

### PR DESCRIPTION
**Pull request by mortezafa**

### CODE REVIEW

**Review by IsaiahA21 — APPROVED** [review]
Looks good to me

File: src/components/TeamSection/TeamInformation.ts [hunk 1]
```diff
@@ -31,11 +31,21 @@ export const executiveTeam: TeamMember[] = teamData.executiveTeam.map((exec, ind
     image: importImage(exec.imagePath)
   }));
 
-export const projectManagers: TeamMember[] = teamData.projectManagers.map((pm, index) => ({
-  id: index,
-  ...pm,
-  image: importImage(pm.imagePath)
-}));
+export const projectManagers: TeamMember[] = teamData.projectManagers.length > 0 ?
```
Comment (Anidion) [inline src/components/TeamSection/TeamInformation.ts:34]: This is a bit shorter and lets `projectManagers` be undefined without breaking
```suggestion
export const projectManagers: TeamMember[] = teamData.projectManagers?.length ? 
```

**Review by Anidion — APPROVED** [review]
Great work

**Review by ParsaKargari — APPROVED** [review]
Looks good, just don't forget to sign off commits in the future - such that DCO doesn't yell at you

**Pull request merged**